internal/sets/match_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sets
+
+import (
+	"testing"
+)
+
+func TestElementsMatch(t *testing.T) {
+	testCases := []struct {
+		listA    []string
+		listB    []string
+		expected bool
+	}{
+		{nil, nil, true},
+		{[]string{"1"}, nil, false},
+		{[]string{"1"}, []string{"1"}, true},
+		{[]string{"1", "2", "1"}, []string{"1", "1", "2"}, true},
+		{[]string{"1", "3", "1"}, []string{"1", "1", "2"}, false},
+	}
+
+	for _, testCase := range testCases {
+		result := StringElementsMatch(testCase.listA, testCase.listB)
```

Comment:
let's also call this function with arguments reversed:

sameResult := StringElementsMatch(testCase.listB, testCase.listA)

if result != sameResult {
  t.Errorf("expected %v to be same as %v", result, sameResult)
}

Reply:
done